server/index.js

```diff
@@ -50,58 +55,98 @@ app.post('/moviedata', (req, res) => {
     .catch(err => console.error('Error posting dummy movie data', err));
 });
 
-/*
-SESSION DATA INPUT:
-{
-  userId: 534356757834,
-  groupId: 1,
-  recs: 1.0,
-  nonRecs: 0.7
-}
-*/
-app.post('/sessionData', (req, res) => {
-  log({ action: 'post request /sessionData' });
-  req.on('error', () => {
-    log({ action: 'post request /sessionData', error: true });
-  });
-  if (!req.body.userId || !req.body.groupId || !req.body.recs || !req.body.nonRecs) {
-    res.sendStatus(400);
-  } else {
-    sessionData(req.body)
-      .then(() => {
-        res.sendStatus(201);
-      })
-      .catch((err) => {
-        log({ action: 'post request /userData', error: true });
-        console.error('Error posting session data', err);
-      });
-  }
-});
+const receiveMessages = options => (
```

Comment:
How about using promisifyAll on all sqs methods?

Reply:
That's a good suggestion!